restaurants/src/app/components/restaurantdetails/restaurantdetails.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component, Input, OnInit } from "@angular/core";
+import { Restaurant } from "../../models/restaurants.model";
+import { RestaurantsService } from "src/app/services/restaurants.service";
+import { RestaurantsComponent } from "../restaurants/restaurants.component";
+import { Router } from "@angular/router";
+import { ActivatedRoute } from "@angular/router";
+import { Location } from "@angular/common";
+import { Observable} from "rxjs";
+import { switchMap } from "rxjs";
+@Component({
+    selector: 'app-restaurantdetails',
+    templateUrl: './restaurantdetails.component.html',
+    styleUrls: ['./restaurantdetails.component.scss']
```

Comment:
If you update to kebab casing, don't forget to update the references here.

Reply:
done